src/js/components/Tab/doc.js

```diff
@@ -65,11 +65,21 @@ export const themeDoc = {
       },
     }`,
   },
+  'tab.border.disabled.color': {
+    description: 'border color of the Tab when disabled',
+    type: 'string | object',
+    defaultValue: undefined,
+  },
   'tab.color': {
     description: 'text color for the Tab.',
     type: 'string',
```

Comment:
while you are already here :)
```suggestion
    type: 'string | {dark: string, light: sting}',
```